server/utils/nunjucksSetup.ts

```diff
@@ -20,9 +20,11 @@ export default function nunjucksSetup(app: express.Express): void {
 
   app.locals.asset_path = '/assets/'
   app.use((req, res, next) => {
-    const externalUser = () => req.url.startsWith('/link') || req.url.startsWith('/barcode')
-    app.locals.applicationName = externalUser() ? 'Send legal mail to prisons' : 'Check Rule 39 mail'
-    app.locals.gtmContainerId = externalUser() ? config.slmContainerId : config.checkRule39ContainerId
+    const externalUser =
+      req.url.startsWith('/link') || req.url.startsWith('/barcode') || req.url.startsWith('/contact-helpdesk')
+    app.locals.externalUser = externalUser
```

Comment:
Set a var as to whether its an external user or not - will be used in `layout.njk` later 👇